tests/resources/VCH-Util.robot

```diff
@@ -72,6 +72,7 @@ Set Test Environment Variables
     ${domain}=  Get Environment Variable  DOMAIN  ''
     Run Keyword If  $domain == ''  Set Suite Variable  ${vicmachinetls}  --no-tlsverify
     Run Keyword If  $domain != ''  Set Suite Variable  ${vicmachinetls}  --tls-cname=*.${domain}
+    Set Environment Variable  VICMACHINETLS  ${vicmachinetls}
```

Comment:
not sure why you are doing this? it is set as a suite variable so should be accessible throughout your test already.  I would have preferred env var from the beginning but I don't think now is the time to change this, and it creates a confusing mixed config as well.

Reply:
It is set as a suite variable? I did not think I was able to access `${vicmachinetls}` where I was using the variable but perhaps I am mistaken. If I am unable to use that variable, I'll get back to you on this. I didn't make this change arbitrarily.